components/rangeslider/rangesliders/twoHandlesSliders/twoHandlesRangeSliderBase.js

```diff
@@ -195,15 +195,43 @@ export default class TwoHandlesRangeSliderBase extends RangeSliderBase {
             distance.indexOf(Math.min(...distance));
     }
 
+    /**
+     * Make a handle active by setting the correct class names
+     * @param {HTMLElement} handle
+     */
+    setActiveHandle(handle) {
+        handle.classList.add('guic-rangeslider-handle-active');
+        handle.classList.remove('guic-rangeslider-handle-inactive');
+    }
+
+    /**
+     * Make a handle inactive by setting the correct class names
+     * @param {HTMLElement} handle
+     */
+    setInactiveHandle(handle) {
+        handle.classList.remove('guic-rangeslider-handle-active');
+        handle.classList.add('guic-rangeslider-handle-inactive');
+    }

```

Comment:
It is OK like that but what if we add 'active' attribute to the active handle and remove/add it to the handles when are focused? Like that you won't take care for two classes at the same time ('guic-rangeslider-handle-active', and 'guic-rangeslider-handle-inactive').  
Also the CSS could be
```
[active] {
    border-style: solid;
    border-width: 2px;
    border-color: #0085a7;
    z-index: 10;
}
```

and the default style of the handle be 
```
    border: none;
    z-index: 5;
```

Reply:
Just think about it, it is not necessary to do it if you don't want to :D

Reply:
It is a lot better and it works in cohtml - changed it.